_posts/2015-04-18-fsharp-core-notes.md

```diff
@@ -75,7 +75,23 @@ If you are using an old-style project file, see the information further below.
 
 If using new-style .NET SDK project files, use:
 
-    <PackageReference Update="FSharp.Core" Version="4.2.3" />
+    <PackageReference Update="FSharp.Core" Version="4.5.0" />
+
+### Make your FSharp.Core references explicit
+
+Modern .NET SDK-style F# projects allow for an _implicit_ FSharp.Core reference, where the `FSharp.Core` package referenced is determined by the .NET SDK being used to compile the project.  The exact version referenced will depend on your .NET SDK, e.g. see [this table](https://github.com/fsharp/fsharp.github.io/pull/89#issuecomment-426729015).
+
+Implicit FSharp.Core references only apply to F# projects and not C# projects. 
```

Comment:
> Nit: they do if you have F# projects in the solution, and if you reference an F# project from a C# project it will respect whatever is specified.

This is not the behaviour I saw if the C# project also references an FSHarp.Core.

Reply:
I suspect the behavior you saw was with our buggy targets that wouldn't respect what you specified.

I just verified today that the C# project will reference what is specified in their project:

```
error NU1605: Detected package downgrade: FSharp.Core from 4.5.2 to 4.3.4. Reference the package directly from the project to s
elect a different version.  [/Users/phillip/scratch/test/test.sln]
```

In this case, the C# project references 4.3.4 and that is respected. Note that in the C# targets, NU1605 is treated as an error rather than a warning; we do not do this in F#.

If I invert this (no C# explicit reference, F# reference a specific version via PackageReference Update) it's also respected, and there is no warning or error.

I just want it to be clear that as of SDK 2.1.400+, whatever you specify _will be respected_. That was not the case with an older SDK you may have used in the past due to a bug that we fixed.